cpython-unix/build.py

```diff
@@ -120,8 +120,6 @@ def add_target_env(env, build_platform, target_triple, build_env):
             target_triple.replace("x86_64_v2-", "x86_64-")
             .replace("x86_64_v3-", "x86_64-")
             .replace("x86_64_v4-", "x86_64-")
-            # TODO should the musl target be normalized?
-            .replace("-unknown-linux-musl", "-unknown-linux-gnu")
```

Comment:
See `xproto` and `xextproto` build changes

Reply:
The answer to this todo seems to be: no it should not generally be normalized like this because it causes problems. But then there are a couple specific builds where it does need to be normalized because there isn't support for `-musl` targets.

See also, the `tcl` build change.